Index: d-feet/data/d-feet.appdata.xml.in
===================================================================
--- /dev/null	1970-01-01 00:00:00.000000000 +0000
+++ d-feet/data/d-feet.appdata.xml.in	2014-09-12 16:49:18.921286457 +0200
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2013 Thomas Bechtold <thomasbechtold@jpberlin.de> -->
+<application>
+  <id type="desktop">d-feet.desktop</id>
+  <licence>CC0</licence>
+  <description>
+    <_p>A simple application for debugging the message bus system D-Bus. D-Feet can be used to inspect D-Bus interfaces of running programs and invoke methods on those interfaces.</_p>
+  </description>
+  <screenshots>
+    <screenshot width="1400" type="default" height="787">https://git.gnome.org/browse/d-feet/plain/data/screenshots/d-feet-overview.png</screenshot>
+  </screenshots>
+  <url type="homepage">https://wiki.gnome.org/DFeet/</url>
+  <updatecontact>thomasbechtold_at_jpberlin_de</updatecontact>
+</application>
