diff -Nur -x '*.orig' -x '*~' d-feet-0.1.6/dfeet/_ui/wnck_utils.py d-feet-0.1.6.new/dfeet/_ui/wnck_utils.py
--- d-feet-0.1.6/dfeet/_ui/wnck_utils.py	2007-12-11 05:01:42.000000000 +0100
+++ d-feet-0.1.6.new/dfeet/_ui/wnck_utils.py	2007-12-18 22:10:21.000000000 +0100
@@ -19,7 +19,7 @@
         self.app_map = {}
 
         icon_theme = gtk.icon_theme_get_default()
-        self.default_icon = icon_theme.load_icon('icon-service', 16, 0)
+        self.default_icon = None
 
         if has_libwnck:
             screen = wnck.screen_get_default()
