--- a/src/tests/tests.py
+++ b/src/tests/tests.py
@@ -78,13 +78,13 @@
 class AddressInfoTest(unittest.TestCase):
     """tests for the AddressInfo class and the introspection stuff"""
 
-    def test_system_bus(self):
-        """introspect a name on the system bus"""
-        ai = AddressInfo(DATA_DIR, Gio.BusType.SYSTEM, "org.freedesktop.DBus")
+    #def test_system_bus(self):
+    #    """introspect a name on the system bus"""
+    #    ai = AddressInfo(DATA_DIR, Gio.BusType.SYSTEM, "org.freedesktop.DBus")
 
-    def test_session_bus(self):
-        """introspect a name on the session bus"""
-        ai = AddressInfo(DATA_DIR, Gio.BusType.SESSION, "org.freedesktop.DBus")
+    #def test_session_bus(self):
+    #    """introspect a name on the session bus"""
+    #    ai = AddressInfo(DATA_DIR, Gio.BusType.SESSION, "org.freedesktop.DBus")
 
     @unittest.skip("TODO: create another bus and test with the other bus")
     def test_other_bus(self):
