Description: Fix crash in tree_model
Author: Robert Ancell <robert.ancell@canonical.com>
Bug: https://bugzilla.gnome.org/show_bug.cgi?id=636024
Index: d-feet-0.1.10/dfeet/introspect_data.py
===================================================================
--- d-feet-0.1.10.orig/dfeet/introspect_data.py	2010-04-19 13:59:03.454051568 +1000
+++ d-feet-0.1.10/dfeet/introspect_data.py	2010-04-19 13:58:57.724039241 +1000
@@ -51,6 +51,8 @@
         return self.child_list.index(child)
 
     def on_get_iter(self, path):
+        if path[0] >= len(self.child_list):
+            return None
         op = self.child_list[path[0]]
         if len(path) == 1:
             return op
