From: Laurent Bigonville <bigon@debian.org>
Date: Tue, 28 Jun 2016 13:29:24 +0200
Subject: Run the tests with python3

Forwarded: no
---
 src/tests/tests.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/tests/tests.py b/src/tests/tests.py
index 1fb5359..0d6b2f9 100755
--- a/src/tests/tests.py
+++ b/src/tests/tests.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
 import sys
 import os
 sys.path.insert(0, os.path.abspath(os.path.join(__file__, "../../")))
