From: Thomas Bechtold <thomasbechtold@jpberlin.de>
Date: Sat, 26 Sep 2015 10:59:58 +0200
Subject: Ignore pep8 E402 error

Bug: https://bugzilla.gnome.org/show_bug.cgi?id=754312
Origin: upstream, 0.3.11, commit:2c430279714a9819e1340477b8c9267e58897529
Bug-Debian: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=797398
---
 src/tests/Makefile.am | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/tests/Makefile.am b/src/tests/Makefile.am
index 7437e52..d8c2f03 100644
--- a/src/tests/Makefile.am
+++ b/src/tests/Makefile.am
@@ -9,4 +9,4 @@ EXTRA_DIST = tests.py
 PEP8_EXCLUDES=--exclude='.svn,CVS,.bzr,.hg,.git,__pycache__,.\#*'
 
 check-local:
-	pep8 --max-line-length=99 --ignore=E123 $(PEP8_EXCLUDES) $(top_srcdir)/src/
+	pep8 --max-line-length=99 --ignore=E123,E402 $(PEP8_EXCLUDES) $(top_srcdir)/src/
